### ISSUE TITLE
Training Loss is Zero ?

### ISSUE THREAD

**Issue by yakhyo** [tsingqguo/exposure-fusion-shadow-removal#33]
Hi, I have a problem with training. I organized the files as mentioned in README file but the training loss is not updating or changing during training, it's always **zero**.

**Comment by fl82hope** [comment]
Could you provide more details, or just debug to see whether the images are read correctly?

**Comment by yakhyo** [comment]
I am using Refine model and checked the dataloaders and datasets as well but could not find the reason behind of this error.

**Comment by yakhyo** [comment]
I downloaded the **ISTD+** dataset and found out that it's litte bit different than this. I could not find `train_C_fixed_official` folder instead of that there is `train_C_fixed_ours`? are these two folders identical ? I tried to train the model changing the folder name from `train_C_fixed_ours`  to `train_C_fixed_official` but `loss` is still **zero**
```
--ISTD+
   --train
      --train_A
          --1-1.png
      --train_B
          --1-1.png 
      --train_C_fixed_official 
          --1-1.png
      --train_params_fixed  # generate later
          --1-1.png.txt
   --test
      --test_A
          --1-1.png
      --test_B
          --1-1.png
      --test_C
          --1-1.png
      --mask_threshold   # generate later
          --1-1.png
```

**Comment by yakhyo** [comment]
1. I am trying to train Refine model, I did not change anything except `n` in `train options` to `5` but still getting **loss** equals to **zero**.
![Screenshot from 2021-11-24 13-35-52](https://user-images.githubusercontent.com/28424328/143175734-c33afe44-81cf-45aa-82a3-7a4f84ac77cc.png)

2. I tested the model according to the weights you provided but the results are not good all of them. Is it normal? or am I doing something wrong? These are samples of testing output:
<div align="center">
<img src="https://user-images.githubusercontent.com/28424328/143176096-f3a8df94-8bf0-46dd-bd1e-dbfc044bcb51.png">
<img src="https://user-images.githubusercontent.com/28424328/143176105-7eeb1b44-1d28-4026-9a78-d55636bb3f06.png">
</div>
